doc/src/cylc-user-guide/cug.tex

```diff
@@ -1203,6 +1203,80 @@ \subsection{Locating Suite And Task Output}
     such as \lstinline=$CYLC_SUITE_SHARE_DIR=, see~\ref{TaskExecutionEnvironment}.
 \end{myitemize}
 
+\subsection{Viewing Suite Logs via Web Browser: Cylc Review}
+
+Cylc provides a utility for viewing the status and logs of suites called
+Cylc Review. Cylc Review displays suite information in web pages, as shown in
+Figure~\ref{fig-review-screenshot}.
+
+\begin{figure}
+    \begin{center}
+        \includegraphics[width=0.5\textwidth]{graphics/png/orig/cylc-review-screenshot.png}
+    \end{center}
+    \caption{Screenshot of a Cylc Review web page}
+\label{fig-review-screenshot}
+\end{figure}
+
+If a Cylc Review server is provided at your site, you can open the Cylc
+Review page for a suite by running the \lstinline=cylc review= command.
+See~\ref{HostsforCylcReview} for requirements and~\ref{ConfiguringCylcReview}
+for configuration steps for setting up a host to run the Cylc Review service
+at your site.
+
+Otherwise an ad-hoc web server can be set up using the
+\lstinline=cylc review start= command argument.
+
+\subsubsection{Hosts For Running Cylc Review}
+\label{HostsforCylcReview}
+
+Installation requirements:
+
+\begin{myitemize}
+
+\item Bash
+\item Python
+\item cherrypy
+\item Jinja2.
+
+\end {myitemize}
+
```

Comment:
This probably should be just an comment on `cylc review`-specific requirements in the CUG installation section.

Reply:
Yes, that is a much better location for it. Sorry for missing that.